distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFtp.java

```diff
@@ -95,61 +113,179 @@ public void afterExam() throws Exception {
+        FTPClient client = createInsecureClient();
 
-        ftpPut(SAMPLE_DATA);
+        ftpPut(client, SAMPLE_DATA);
 
         // verify FTP PUT resulted in ingest, catalogued data
         Response response = executeOpenSearch("xml", "q=*", "count=100");
-        response.then().log().all().body("metacards.metacard.size()", equalTo(1));
+        response.then()
+                .log()
+                .all()
+                .body("metacards.metacard.size()", equalTo(1));
 
         // clean up test data
         TestCatalog.deleteMetacard(getMetacardIdFromResponse(response));
+        disconnectClient(client);
     }
 
     /**
      * Upload a file via insecure FTP for ingest using streaming method
-     * 
+     *
      * @throws Exception
      */
     @Test
-    public void testInsecureFtpPutStream() throws Exception {
-        client = createInsecureClient();
+    public void testFtpPutStream() throws Exception {
+        setClientAuthConfiguration(CLIENT_AUTH_WANT);
+        FTPClient client = createInsecureClient();
 
         StringWriter writer = new StringWriter();
         IOUtils.copy(getClass().getResourceAsStream("/metacard1.xml"), writer);
-        ftpPutStreaming(writer.toString());
+        ftpPutStreaming(client, writer.toString());
 
         // verify FTP PUT resulted in ingest, catalogued data
         Response response = executeOpenSearch("xml", "q=*", "count=100");
-        response.then().log().all().body("metacards.metacard.size()", equalTo(1));
+        response.then()
+                .log()
+                .all()
+                .body("metacards.metacard.size()", equalTo(1));
 
         // clean up test data
         TestCatalog.deleteMetacard(getMetacardIdFromResponse(response));
+        disconnectClient(client);
+    }
+
+    /**
+     * Upload a file via FTPS for ingest using input stream
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testFtpsPut() throws Exception {
+        setClientAuthConfiguration(CLIENT_AUTH_NEED);
+        FTPSClient client = createSecureClient(true);
+
+        ftpPut(client, SAMPLE_DATA);
+
+        // verify FTP PUT resulted in ingest, catalogued data
+        Response response = executeOpenSearch("xml", "q=*", "count=100");
+        response.then()
+                .log()
+                .all()
+                .body("metacards.metacard.size()", equalTo(1));
+
+        // clean up test data
+        TestCatalog.deleteMetacard(getMetacardIdFromResponse(response));
+        disconnectClient(client);
+    }
+
+    /**
+     * Upload a file via FTPS for ingest using streaming method
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testFtpsPutStream() throws Exception {
+        setClientAuthConfiguration(CLIENT_AUTH_NEED);
+        FTPSClient client = createSecureClient(true);
+
+        StringWriter writer = new StringWriter();
+        IOUtils.copy(getClass().getResourceAsStream("/metacard1.xml"), writer);
```

Comment:
This also reminds me. Should we be doing `TestFtp.class.getClassLoader().getResourceAsStream()` here instead, for windows compatibility. Calling @Lambeaux